core/assets/bundles/bundle_es.properties

```diff
@@ -1526,7 +1526,7 @@ liquid.ozone.name = Ozono
 liquid.hydrogen.name = Hidrógeno
 liquid.nitrogen.name = Nitrógeno
 liquid.cyanogen.name = Cianógeno
-
+#Names of Units and Turrets looks better untranslated, since they are names
```

Comment:
Note that comments get automatically removed afterwards to sync with the main bundle

Reply:
Done in 8eec04774f82dea6a76959401996cc71ff2971f4